hr_employee_birth_name/readme/DESCRIPTION.rst

```diff
@@ -0,0 +1,3 @@
+This module adds the following to employees :
```

Comment:
Useless description, what this module does?

Reply:
Enable a field to add the birth name of an employee.

Reply:
"This module adds feature to provide birth name of an employee"